config/initializers/scheduler.rb

```diff
@@ -0,0 +1,12 @@
+require 'rufus-scheduler'
+
+s = Rufus::Scheduler.singleton
+
+return if defined?(Rails::Console) || Rails.env.test? || File.split($0).last == 'rake'
+
+# Send weekly news updates on Friday 5:30pm
+s.cron '30 17 * * 5' do
+  RubySgBotSubscriber.in_batches.each_record do |sub|
+    SendWeeklyUpdatesJob.perform_now(sub.chat_id)
+  end
```

Comment:
I am a bit confused here. `RubySgBotSubscriber` contains all the subscribers (or members) of the Telegram group right? So in this case, this loop sends a `SendWeeklyUpdates` to each member?

Reply:
Subscribers are per chat basis. RubySg group is considered 1 subscriber. So the message will be pushed to the group instead of sending to each individual members inside the group.